### ISSUE TITLE
Pamac 4.1.0  UI issue

### ISSUE THREAD

**Issue by MonroOnManjaro** [manjaro/pamac#130]
I'm running manjaro-i3-16.06-rc-x86_64 under virtualbox, after the latest updates.

Clicking on each package on the package manager open the details pane.
I am trying to install several packages and this is a usage-annoyance.

I think the details pane should be brought back to its former place as in 3.2.1, that is as a lower part of the list (similar to reading pane on web email pages).
That way it is both visible if needed and not requiring extra clicks.

**Comment by philmmanjaro** [comment]
@MonroOnManjaro: this is a wanted feature with **v4.x**. See also https://github.com/manjaro/pamac/issues/113 https://github.com/manjaro/pamac/issues/126

**Comment by MonroOnManjaro** [comment]
I understand the need for quick access to details. I asked about it myself in my other issue (#123). 
That pane with all its tabs is frequently needed.
But the current "pane-jump" (for lack of better term) implementation is both user-unfriendly, and disturbs productivity.
IMHO, the implementation should be reconsidered.

**Comment by maxrot** [comment]
I have an idea why don't we make a survey or a poll to see how many users like the changes. Vote on features to implement or not on the homepage. My problem is in fact that the reinstall option was taken away from the right click menu. I use it allot I don't want click on every one to just click reinstall. But good things have also came out of my not liking of these changes. I found Pamac a burden to use. So I learned more about Pacman and have been using it more and more. But on the positive side allot is good, it works quicker I like the fact that I can use the package manager when I install. I don't prefer CSD, but I will except it with only Pamac! I have no other CSD apps. I would also like in the descriptions section if there was a screenshot of the application so the users could see what the program looks like. In the options section it would be cool to add a selection for stable testing or unstable. The question is always wanted by how many users! I hate the mouse it is evil it should be should be used as less as possible LOL!
